### PR TITLE
Changed owners:

### PR DESCRIPTION
**Pull request by nirhutnik**
* Added Noam to approve docs
* Changed that devs are only responsible for tests/base instead of tests/ folder. This way, most changes in checks will not require devs (and can be either algo or devs).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 *       @deepchecks/deepchecks-team
 
-/docs/ @ItayGabbay @shir22
+/docs/ @ItayGabbay @shir22 @noambzr
```
Comment (ItayGabbay) [inline .github/CODEOWNERS:9]: It doesn't work. I think it's noamzbr?

**Review by ItayGabbay — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (nirhutnik) [reply .github/CODEOWNERS]: You're right, thanks!

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -15,7 +15,7 @@ README.rst @shir22
 
 requirements/ @ItayGabbay
 deepchecks/core/ @deepchecks/dev-team
-tests/ @deepchecks/dev-team
+tests/base @deepchecks/dev-team
```
Comment (noamzbr) [inline .github/CODEOWNERS:18]: Add also tests/serialization

Comment (nirhutnik) [reply .github/CODEOWNERS:18]: Fixed

**Review by noamzbr — APPROVED** [review]

**Review by ItayGabbay — APPROVED** [review]

**Pull request merged**